package.json

```diff
@@ -4,13 +4,29 @@
   "description": "A textlint plugin for LaTeX2e",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "eslint src/**/*.ts",
+    "lint": "run-p lint:*",
+    "lint:eslint": "eslint ./**/*.ts",
+    "lint:prettier": "prettier --check ./**/*.ts",
+    "fix": "run-s fix:prettier fix:eslint",
+    "fix:eslint": "eslint --fix ./**/*.ts",
+    "fix:prettier": "prettier --write ./**/*.ts",
     "build": "tsc",
     "test": "jest",
     "clean": "rimraf lib",
     "prepare": "npm run clean && npm run build",
     "prepublishOnly": "npm test"
   },
+  "lint-staged": {
+    "*.ts": [
+      "prettier --write",
+      "eslint --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
```

Comment:
For lint-staged and husky, we can use `lintstaged.config.js` and `husky.config.js`.
The large package.json is undesirable according to KISS principle.